File: hub-kit-backend/src/main/java/com/inetum/realdolmen/hubkitbackend/dto/TrailerDTO.java
package com.inetum.realdolmen.hubkitbackend.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class TrailerDTO extends VehicleDTO {
    private Boolean hasRegistration;
    private String ofVehicle;
}

File: crash-kit-app/app/src/main/java/com/inetum/realdolmen/crashkit/dto/TrailerDTO.kt
package com.inetum.realdolmen.crashkit.dto

data class TrailerDTO(
    @Transient override val id: Int?,
    @Transient override val licensePlate: String?,
    @Transient override val countryOfRegistration: String?,
    val hasRegistration: Boolean,
    val ofVehicle: String?
) : Vehicle(id, licensePlate, countryOfRegistration)
